ui/app/src/business/services/DataService/DataService.ts

```diff
@@ -22,15 +24,44 @@ export type RewardsProgram = {
 };
 
 export type UserRewardsSummaryResponse = {
-  totalClaimableCommissionsAndClaimableRewards: number;
-  maturityDateMs: string;
-  yearsToMaturity: number;
-  totalDepositedAmount: number;
-  currentTotalCommissionsOnClaimableDelegatorRewards: number;
-  claimedCommissionsAndRewardsAwaitingDispensation: number;
-  dispensed: number;
-  totalCommissionsAndRewardsAtMaturity: number;
-}[];
+  reward_program: string;
+  pool: string;
+  net_liquidity_bal: number;
+  total_liquidity_bal: number;
+  net_percentage: number;
+  reward_allocation: number;
+  pool_unit: string;
+  total_pool: string;
+  perc_pool: 0.00010346942052193494;
+};
+
+type ProgramConfigMap = Record<
+  string,
+  {
+    displayName: string;
+    description: string;
+    documentationURL: string;
+    summaryAPY: number;
+  }
+>;
+
+export const REWARDS_PROGRAMS_CONFIG: ProgramConfigMap = {
```

Comment:
I think these should be returned from the rewardConfig endpoint.  I will add them next week.